pkg/templates/templates.go

```diff
@@ -0,0 +1,28 @@
+package templates
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/giantswarm/microerror"
+)
+
+func Render(templates []string, data interface{}) (string, error) {
+	var err error
+
+	main := template.New("main")
+	for _, t := range templates {
```

Comment:
What's the use-case for a list of templates here? Isn't it that last one wins (overrides prior ones) anyway?